ocw/lib/db.py

```diff
@@ -224,8 +224,8 @@ def auto_delete_instances():
                          age__gte=F('ttl')).exclude(csp_info__icontains='pcw_ignore')
         email_text = set()
         for i in obj:
-            logger.info("[%s] TTL expire for instance %s:%s %s", i.vault_namespace,
-                        i.provider, i.instance_id, i.all_time_fields())
+            logger.debug("[%s] TTL expire for instance %s:%s %s", i.vault_namespace,
```

Comment:
I would keep this log as `info`.

Reply:
one line below there is delete_instance call which will gives you notification about deletion of entity anyway , now we getting 2 lines per one deletion it is too much